routers/websocket.py

```diff
@@ -0,0 +1,83 @@
+"""
+Regestration router
```

Comment:
http mà nhỉ, sao lại đặt trong websocket.py . Ngoài ra chắc để anh viết vài cái để em follow theo. Chứ a ko muốn đặt tên thế này

Reply:
okieee anh em cũng tính hỏi anh có đặt tên z không nên chờ anh cmt vào đó

Reply:
`- .gitignore
- app.py # main fastapi 
- config/ # config
  - __init__.py
  - log-config.yml
  - secret.toml
- endpoints/ # connect others services
  - __init__.py
  - mongo.py
- models/ # models for DB 
  - users.py
- routers/ # routers : api
  - __init__.py
  - admin.py
  - users.py
- schemas/ # Basemodel for Pydantic
  - __pycache__/
  - users.py
- utils/
  - security.py
- venv/
- Readme.md
- reqs.tx`